### PR TITLE
Add CC-News dataset of English language articles

### PR DESCRIPTION
**Pull request by vblagoje**
Adds [CC-News](https://commoncrawl.org/2016/10/news-dataset-available/) dataset. It contains 708241 English language news articles. Although each article has a language field these tags are not reliable. I've used Spacy language detection [pipeline](https://spacy.io/universe/project/spacy-langdetect) to confirm that the article language is indeed English. 

The prepared dataset is temporarily hosted on my private Google Storage [bucket](https://storage.googleapis.com/hf_datasets/cc_news.tar.gz). We can move it to HF storage and update this PR before merging.

### CODE REVIEW

**Comment by tanmoyio** [comment]
@vblagoje nice work, please add the README.md file and it would be ready

**Comment by vblagoje** [comment]
@lhoestq @tanmoyio @yjernite please have a look at the dataset card. Don't forget that the dataset is still hosted on my private gs bucket and should be moved to HF buckets

File: datasets/cc_news/README.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 100K<n<1M
+source_datasets:
+- original
+task_categories:
+- unknown
+task_ids:
+- unknown
```
Comment (lhoestq) [inline datasets/cc_news/README.md:19]:
```suggestion
task_categories:
- sequence-modeling
task_ids:
- language-modeling
```

maybe add the language modeling task ?

**Review by lhoestq — COMMENTED** [review]
Looking pretty good so far thanks !

File: datasets/cc_news/README.md (hunk 1 above)
Comment (vblagoje) [reply datasets/cc_news/README.md]: Great idea, updated!

**Comment by lhoestq** [comment]
I will move the files soon and ping you when it's done and with the new URLs :)

**Comment by lhoestq** [comment]
Hi !

I just moved the file to a HF bucket. It's available at https://storage.googleapis.com/huggingface-nlp/datasets/cc_news/cc_news.tar.gz

**Comment by vblagoje** [comment]
@lhoestq no worries, updated PR with the new URL and rebased to master

File: datasets/cc_news/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+### Languages
+
+The text in the dataset is in the English language. 
+
+## Dataset Structure
+
+### Data Instances
+
+Dataset instance contains an article itself and the relevant article fields.
+An example from the Cc-New test set looks as follows:
```
Comment (lhoestq) [inline datasets/cc_news/README.md:72]:
```suggestion
An example from the Cc-New train set looks as follows:
```
there's only a train set

**Review by lhoestq — APPROVED** [review]
Awesome thank you !

Looks all good now :)

**Pull request merged**